src/telemetry/metrics.ts

```diff
@@ -10,26 +10,30 @@ import { GcpDetectorSync } from '@google-cloud/opentelemetry-resource-util';
 import { isProd } from '../common';
 
 export const startMetrics = (serviceName: string): void => {
-  const metricReader = isProd
-    ? new metrics.PeriodicExportingMetricReader({
```

Comment:
Why didn't we use prometheus exporter before in production?

Reply:
Because we didn't have prometheus in production before. Added it a few weeks ago, and as it totes that it's cheaper metrics than gcloud, it's worth a look.

Reply:
Ok, if it is not super urgent we can also merge it after the weekend to avoid any issues with current metrics.

Reply:
It won't affect regular metrics, as we can have multiple metric exporters.
So it will both do the old one we used, and add prometheus metrics

Reply:
Yes I see, but I am not 100% on what can go wrong so that is why I am saying.

Reply:
I'll keep an eye out, and revert it if I see it causes any issues. I was planning on playing around with the prometheus this weekend 😬